__tests__/app.test.js

```diff
@@ -140,3 +138,46 @@ describe("Get /api/articles", () => {
         })
     })
 })
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("200: Should respond with an array of comments for the given article_id with the relevant properties, ordered by most recent comment first", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+                for (let i = 0; i < body.comments.length; i++){
+                expect(body.comments[i].article_id).toBe(1)
+                expect(body.comments.length).toBeGreaterThan(0)
+            }
+            body.comments.forEach((item) => {
+                expect(item).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String)
+                })
+                expect(body.comments.length).toBeGreaterThan(0)
+                
+            })
+            for (let i = 0; i < body.comments.length -1; i++){
+                let latest = new Date(body.comments[i].created_at);
+                let next = new Date(body.comments[i + 1].created_at);
+                expect(next.getTime()).toBeLessThanOrEqual(latest.getTime())
+                expect(body.comments.length).toBeGreaterThan(0)
+            }
+        })
+    })
+    test("200: Should respond with an array of comments, ordered by date created for a different article_id", () => {
+        return request(app)
+        .get("/api/articles/5/comments")
+        .expect(200)
+        .then(({ body }) => {
+            for (let i = 0; i < body.comments.length; i++){
+            expect(body.comments[i].article_id).toBe(5)
+            expect(body.comments.length).toBeGreaterThan(0)
+           }
```

Comment:
If we have asserted in our description that we are specifying an order then we need to include an assertion for it too. The npm package jest sorted will be handy here. https://www.npmjs.com/package/jest-sorted?activeTab=readme